src/popup/components/icon.tsx

```diff
@@ -91,11 +99,11 @@ export function IconLink(props: IconProps & LinkProps): JSX.Element {
 }
 
 function iconProps<T>(props: T & IconProps): IconProps {
-  return pick(props, ['title', 'svg', 'size', 'theme'])
+  return pick(props, ['title', 'svg', 'size', 'theme', 'className'])
 }
 
 function nonIconProps<T extends object & IconProps>(
   props: T,
 ): Omit<T, keyof IconProps> {
-  return omit(props, ['title', 'svg', 'size', 'theme'])
+  return omit(props, ['title', 'svg', 'size', 'theme', 'className'])
```

Comment:
This feels a little shady because the `<Link>`, `<button>` and `<Icon>` all have a `className` prop but here we specifically give it to the icon.

Honestly, the more I think about it, the more I think that the link and button variants for the icons might have been a bad idea.